packages/terra-form-select/src/terra-dev-site/doc/form-select/ComboboxField.06.doc.jsx

```diff
@@ -27,13 +29,17 @@ const DocPage = () => (
       },
     ]}
     examples={[{
-      title: 'Combobobx Field Example',
+      title: 'Combobox Field Example',
       example: <ComboboxFieldExample />,
       source: ComboboxFieldExampleSrc,
     }, {
       title: 'Required Combobobx Field Example',
       example: <RequiredComboboxFieldExample />,
       source: RequiredComboboxFieldExampleSrc,
+    }, {
+      title: 'Incomplete Combobobx Field Example',
```

Comment:
combobobx ➡️combobox, throughout this file.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2670/commits/2364951248c995daf9fb032d47c5a2d367d8bf38